### DISCUSSION TITLE
Srt file?

### DISCUSSION THREAD

**Discussion by Qel0droma** [SYSTRAN/faster-whisper#186] [Q&A]
hello, i have installed this and have be giving it a shot and loving how much faster it is and the less vram usage is amazing. though it doesn't seem to make an srt file like the orginal whisper? how do i get it to output an srt file while running the command from Powershell? thanks

**Comment by Qel0droma** [comment]
i have tried with "whisper-ctranslate2 "test.mp4" --device cuda --device_index 0 --vad_filter true --vad_min_speech_duration_ms 50 --vad_min_silence_duration_ms 2000 --vad_max_speech_duration_s 5 --condition_on_previous_text False --language Japanese --task translate --model large-v2 --output_format srt" no srt file is outputted. :(

**Reply by guillaumekln** [reply]
You should look for help in the whisper-ctranslate2 repo since this is where the SRT file should be written.
